packages/patternfly-4/react-core/src/demos/PaginationTable.md

```diff
@@ -66,3 +68,140 @@ class ComplexPaginationTableDemo extends React.Component {
+  constructor(props) {
+    this.columns = [
+      { title: "First column" },
+      { title: "Second column" },
+      { title: "Third column" }
+    ];
+    this.defaultRows = [
+      { cells: [
+        { title: "Row 1 column 1" },
+        { title: "Row 1 column 2" },
+        { title: "Row 1 column 3" }
+      ]},
+      { cells: [
+        { title: "Row 2 column 1" },
+        { title: "Row 2 column 2" },
+        { title: "Row 2 column 3" }
+      ]},
+      { cells: [
+        { title: "Row 3 column 1" },
+        { title: "Row 3 column 2" },
+        { title: "Row 3 column 3" }
+      ]},
+      { cells: [
+        { title: "Row 4 column 1" },
+        { title: "Row 4 column 2" },
+        { title: "Row 4 column 3" }
+      ]},
+      { cells: [
+        { title: "Row 5 column 1" },
+        { title: "Row 5 column 2" },
+        { title: "Row 5 column 3" }
+      ]},
+      { cells: [
+        { title: "Row 6 column 1" },
+        { title: "Row 6 column 2" },
+        { title: "Row 6 column 3" }
+      ]},
+      { cells: [
+        { title: "Row 7 column 1" },
+        { title: "Row 7 column 2" },
+        { title: "Row 7 column 3" }
+      ]},
+      { cells: [
+        { title: "Row 8 column 1" },
+        { title: "Row 8 column 2" },
+        { title: "Row 8 column 3" }
+      ]},
+      { cells: [
+        { title: "Row 9 column 1" },
+        { title: "Row 9 column 2" },
+        { title: "Row 9 column 3" }
+      ]},
+      { cells: [
+        { title: "Row 10 column 1" },
+        { title: "Row 10 column 2" },
+        { title: "Row 10 column 3" }
+      ]},
+      { cells: [
+        { title: "Row 11 column 1" },
+        { title: "Row 11 column 2" },
+        { title: "Row 11 column 3" }
+      ]},
+      { cells: [
+        { title: "Row 12 column 1" },
+        { title: "Row 12 column 2" },
+        { title: "Row 12 column 3" }
+      ]}
+    ];
+    this.defaultPerPage = 10;
+    this.state = {
+      perPage: this.defaultPerPage,
+      page: 1,
+      rows: this.defaultRows.slice(0, this.defaultPerPage)
+    };
+  }
+
+  handleSetPage(page, startIdx, endIdx) {
+    this.setState({
+      page,
+      rows: this.defaultRows.slice(startIdx, endIdx)
+    });
+  }
+
+  handlePerPageSelect(page, perPage, startIdx, endIdx) {
+    this.setState({
+      perPage,
+      page,
+      rows: this.defaultRows.slice(startIdx, endIdx)
+    });
+  }
+
+  renderPagination(variant = 'top') {
+    const { page, perPage, total } = this.state;
+    return (
+      <Pagination
+        itemCount={this.defaultRows.length}
+        page={page}
+        perPage={perPage}
+        defaultToFullPage
+        onSetPage={(_evt, newPage, perPage, startIdx, endIdx) => {
+          this.handleSetPage(newPage, startIdx, endIdx);
+        }}
+        onPerPageSelect={(_evt, newPerPage, newPage, startIdx, endIdx) => {
+          this.handlePerPageSelect(newPage, newPerPage, startIdx, endIdx);
+        }}
```

Comment:
I think we can simplify this demo by skipping the intermediate anonymous function here and just call `handlePerPageSelect` handler directly. Since handlePerPageSelect isn't used elsewhere there isn't as much value in breaking it out.